app/models/User.coffee

```diff
@@ -180,6 +180,19 @@ module.exports = class User extends CocoModel
 
   isOnPremiumServer: ->
     me.get('country') in ['china', 'brazil']
+    
+  spy: (target, options={}) ->
+    target = target.id or target
```

Comment:
Trying out for these sorts of functions taking a string or model. Tweaked the endpoint during refactoring to also take an ID, so you could hand in an id string or a User model instance. Haven't refactored the name of the property, though, which would make more sense as 'target' or 'user' instead of 'nameOrEmailLower'. I'll tweak that.